lib/valueflow.cpp

```diff
@@ -1345,7 +1345,7 @@ static void valueFlowArray(TokenList *tokenlist)
             // const array decl
             else if (tok->variable() && tok->variable()->isArray() && tok->variable()->isConst() &&
                      tok->variable()->nameToken() == tok && Token::Match(tok, "%var% [ %num%| ] = {")) {
-                const Token* rhstok = tok->next()->link()->tokAt(2);
+                Token* rhstok = tok->next()->link()->tokAt(2);
```

Comment:
I think these can stay `const`.

Reply:
These changes are all based straight on compiler errors when the signature has been fixed - so no, they cannot.

Also from looking at the code it is used to get a new value for `tok` two lines down and that is not `const`.